database/index.js

```diff
@@ -1,11 +1,74 @@
 const initOptions = {
-  schema: 'reviews',
+  schema: process.env.DB_SCHEMA,
 };
-
 const pgp = require('pg-promise')(initOptions);
-const cn = `postgres://sunikkim:${process.env.DB_PASSWORD}@localhost:5432/reviews`;
+
+const helpers = require('./dbHelpers');
+
+const cn = `postgres://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_DATABASE}`;
 const db = pgp(cn);
 
+const getReviews = (productId) => {
+  const query = `
+  SELECT
```

Comment:
Nice query string - I think I can use some of your logic here to help clean up mine!